packages/app/src/cli/models/app/app.ts

```diff
@@ -49,6 +55,13 @@ function removeTrailingPathSeparator(value: string[] | undefined) {
   // eslint-disable-next-line no-useless-escape
   return value?.map((dir) => dir.replace(/[\/\\]+$/, ''))
 }
+
+// If a path ends with a single asterisk, modify it to end with a double asterisk.
+// This is to support the glob pattern used by chokidar and watch for changes in subfolders.
+function fixSingleWildcards(value: string[] | undefined) {
+  return value?.map((dir) => dir.replace(/\*$/, '**'))
```

Comment:
Wouldn't this regex also convert a double-asterisk to a triple-asterisk? I think you might want a captured non-asterisk, followed by an asterisk and then end-of-line, as your regex:

```suggestion
  return value?.map((dir) => dir.replace(/([^\*])\*$/, '$1**'))
```

Reply:
Good catch, this will work for any kind of path type `/` or `\`
I'll add tests to it though :)